spec/support/features/idv_helper.rb

```diff
@@ -96,4 +96,29 @@ def complete_idv_profile_ok(user, password = user_password)
     fill_in 'Password', with: password
     click_submit_default
   end
+
+  def visit_idp_from_sp_with_loa3(state: SecureRandom.hex, client_id:, nonce:)
```

Comment:
Thoughts on finding a way to rename this so it is clear it's specific to OIDC? I'd also think `visit_idp_from_sp_with_loa3` fits better on the method that is currently named `perform_loa3_request_from_sp`.